docs/pages/blog/danilo-leal-joining.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Danilo Leal joins Material-UI
+description: We are excited to share that Danilo Leal has joined Material-UI.
+date: 2021-07-14T00:00:00.000Z
+authors: ['oliviertassinari']
+card: true
+---
+
+We are excited to share that [Danilo Leal](https://daniloleal.co/) has joined Material-UI!
+He started a couple of days ago as a lead designer.
+
+Before joining Material-UI, he worked as a designer at Loggi, a Brazilian logistics unicorn.
+He has spent the last five years diving into product design, product strategy, and design systems.
+Danilo strongly believes that code is a fundamental piece of a designer's work. It is, after all, the real medium through where users experience the product.
+For instance, one of his continuous contributions to Loggi was deep-diving into Material-UI's codebase to fulfill Loggi's customization needs.
+
+His aspirations are well aligned with where we want to push the company in the long term.
+He wants to challenge the status quo.
+He wants to push the envelope around how products are built.
+We believe there are two high potential opportunities to explore: 1. how designers and developers collaborate, 2. how accessible is creating custom UIs.
+His design [challenge assignment](https://www.figma.com/file/UPgXH2mdWqV8QzAuN5VM37/Danilo-assignment) (done under a few hours) gave up confidence that Danilo is exactly who the role needs.
```

Comment:
```suggestion
His design [challenge assignment](https://www.figma.com/file/UPgXH2mdWqV8QzAuN5VM37/Danilo-assignment) (done under a few hours) gave us confidence that Danilo is exactly who the role needs.
``` 
?

Reply:
Oh wow haha. On point correction 😅
"Gave up" changes the meaning completely haha!

Reply:
I hope it’s ‘gave us’ 😁